app/controllers/defence_requests_controller.rb

```diff
@@ -1,8 +1,10 @@
 class DefenceRequestsController < BaseController
 
+  include DefenceRequestAuthorization
+
```

Comment:
as discussed with @FriedSock and @robmckinnon i think this is a good refactor if we can make it generic enough to use across all the controllers